### PR TITLE
Update conda instructions

### PR DESCRIPTION
**Pull request by brahmaneya**
**Test Plan**
Tested spouse and crowd tutorials with new conda instructions on an AWS instance with Deep Learning AMI (ubuntu) p3.2xlarge. spam is failing due to unrelated bug.

### CODE REVIEW

**Comment by henryre** [comment]
Couple of changes requested but approving to unblock

File: README.md [hunk 1]
```diff
@@ -113,17 +113,21 @@ These commands assume that your conda installation is Python 3.6+.
 
 ```bash
 # [OPTIONAL] Activate a virtual environment
-conda create --name spam
+# Python 3.7.4 cause some issues e.g. https://github.com/tensorflow/tensorflow/issues/31523
+conda create --yes -n spam python=3.6 pytorch==1.1.0
```
[the remark this answers is not in the document]
Comment (brahmaneya) [reply README.md]: If you try to install it via pip (say from requirements.txt), it fails with error: "Could not find a version that satisfies the requirement torch>=0=1.1.0,<1.2.0"

File: README.md [hunk 2]
```diff
@@ -113,17 +113,21 @@ These commands assume that your conda installation is Python 3.6+.
...
 source activate spam
 
 # Install requirements (both shared and tutorial-specific)
-conda install --yes --file requirements.txt
-conda install --yes --file spam/requirements.txt
+pip install environment_kernels
```
[the remark this answers is not in the document]
Comment (brahmaneya) [reply README.md]: It's needed for jupyter to detect available kernels; without it, jupyter notebook fails when running from a conda environment (but doesn't fail when running from a regular venv for some reason).

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (henryre) [reply README.md]: hm seems like the ">=0" is a typo somewhere

[the remark this answers is not in the document]
Comment (henryre) [reply README.md]: hm but doesn't look like it's in any of our stuff...

[the remark this answers is not in the document]
Comment (brahmaneya) [reply README.md]: Oh no, that's my typo. It was torch>=1.1.0,<1.2.0

[the remark this answers is not in the document]
Comment (brahmaneya) [reply README.md]: Let me make sure I can reproduce this from aws again. Direct pip install is working on my regular aws instance. It's possible pytorch has temporarily removed a version from pypi, and put it back.

[the remark this answers is not in the document]
Comment (brahmaneya) [reply README.md]: This appears to be fixed now. So I'll remove the conda pytorch install

**Pull request merged**